platformio.ini

```diff
@@ -16,6 +16,7 @@ default_envs =
 #    mppt-1210-hus-v0.7
 #    pwm-2420-lus-v0.2
 #    pwm-2420-lus-v0.3
+    pwm-2420-lus-v0.3-bl
```

Comment:
Please keep the comment here. By default, none of the environments should be selected.

Reply:
Agreed and updated